src/WindowsAzureMessaging/WindowsAzureMessaging/Utils/MSCompression.m

```diff
@@ -0,0 +1,87 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Is it necessary to pull this in for the preview? I know we were trying to avoid adding dependencies, but this seems excessive. Does the built-in HTTP library in iOS not support gzip transparently?